src/views/List.jsx

```diff
@@ -1,8 +1,27 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { ListItem } from '../components/ListItem';
+import { Link } from 'react-router-dom';
 
-export function List({ data }) {
+export function List({ data, listPath }) {
 	const [search, setSearch] = useState('');
+	const [listName, setListName] = useState('');
+
+	const extractAfterSlash = (listPath) => {
```

Comment:
I think if you used this: `const [listName] = useState(listPath.split('/')[1]);` you could get rid of the whole `extractAfterSlash` function and I believe the `useEffect` as well. Would save about 15 lines of code.